### PR TITLE
add optional link to CGU and confidentiality policy

### PR DESCRIPTION
**Pull request by monodo**

### CODE REVIEW

File: templates/two_factor/login.html [hunk 1]
```diff
@@ -25,6 +25,18 @@
 {% block content-fluid %}
   <div id="login-content">
     <div class="login-welcome-text">
+      <div id="landing-page-link">
+        <a href="https://geocity.ch/about" target="_blank" rel="noreferrer">{% trans "Découvrir Geocity" %}</a>
```
Comment (rbovard) [inline templates/two_factor/login.html:29]: Remove it at line 109

File: static/css/login.css [hunk 2]
```diff
@@ -113,12 +107,7 @@ body {
   }
 
   #landing-page-link {
-    display:block;
-    position:absolute;
-    float:center;
-    bottom:10px;
-    left:40%;
-    font-size: 12px;
+    font-size: 0.8rem;
```
Comment (rbovard) [inline static/css/login.css:110]: Maybe smaller to be less visible, no?

**Comment by rbovard** [comment]
@monodo Could please add screechots of dektop and mobile versions to see the result?

**Comment by monodo** [comment]
thanks @rbovard

**Pull request merged**